### PR TITLE
Align source and destination images

### PR DESCRIPTION
**Pull request by opatiny**
- feat: implement affine transform utilities
- test: add ransac tests with 2D points
- feat: start implementing getDestinationOrigin Refs: #248
- fix: affineTransform import issues
- test: fix ransac test
- fix: use ransac in getDestinationOrigin
- feat: implement overlapImages
- fix(overlapImages): colorModel issue
- test: add getAffineTransform test
- fix(overlapImages): remove useless lines
- feat(getAffineTransform): change return object
- fix(getAffineTransform): round translation coordinates
- refactor: move getBrief
- fix(getAffineTransform): works with ID card crops!
- feat(overlapImages): add angle and scale
- docs(copyTo): enhance tsdoc
- fix: aligning works
- test(getAffineFunction): rotation works
- feat: working on script to align images
- fix(alignImages): empty folder with fs.opendir
- fix(alignImages): deleting files works
- chore: minor changes
- feat: implement filterSmallestDistanceMatches
- feat: implement filterEuclidianDistance
- fix: alignImages works even with slightly different typos

### CODE REVIEW

File: src/featureMatching/affineTransfrom/getEuclidianDistance.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { Point } from '../..';
+
+/**
+ * Compute the distance between point 1 and point 2.
+ * @param point1 - First point.
+ * @param point2 - Second  point.
+ * @returns Euclidian distance.
+ */
+export function getEuclidianDistance(point1: Point, point2: Point): number {
```
Comment (targos) [inline src/featureMatching/affineTransfrom/getEuclidianDistance.ts:9]: Euclidean

Comment (opatiny) [reply src/featureMatching/affineTransfrom/getEuclidianDistance.ts:9]: Woops '^^

**Comment by targos** [comment]
Can you point me to what you would use from the outside project? What are the entry points of the API?

**Comment by opatiny** [comment]
> Can you point me to what you would use from the outside project? What are the entry points of the API?

The only useful function will be `getAffineTransform`, which takes in two images and returns the transform from source to destination. Based on the transform, I can compute the translation to align the ROIs directly in PADIF.

I also exposed `overlapImages`, which is a practical debugging tool imo.

**Pull request merged**